package/Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM registry.suse.com/bci/bci-base:15.4
 
 RUN zypper -n rm container-suseconnect && \
+    zypper install -y curl=7.79.1-150400.5.18.1 \
+      glibc=2.31-150300.46.1 \
+      libcurl4=7.79.1-150400.5.18.1 \
+      libopenssl1_1=1.1.1l-150400.7.31.2 \
+      libopenssl1_1-hmac=1.1.1l-150400.7.31.2 \
+      libzstd1=1.5.0-150400.3.3.1 \
+      openssl-1_1=1.1.1l-150400.7.31.2 && \
```

Comment:
IIUC, this will not be needed as the OS will bump those packages automatically. cc @bk201

Reply:
Yes, no need to specify the version. A new build should upgrade those packages.